frontend/device/input.lua

```diff
@@ -1028,14 +1029,17 @@ function Input:waitEvent(now, deadline)
                             -- This is why we clear the full list of timers on the first match ;).
                             self:clearTimeouts()
                             self:gestureAdjustHook(touch_ges)
-                            return Event:new("Gesture",
+                            return { Event:new("Gesture",
```

Comment:
Not sure what to think of the exact spacing/layout.

Reply:
Yeah, possibly break after the brace like we usually do for constructors & stuff?